EIPS/eip-aura.md

```diff
@@ -9,41 +9,74 @@ created: 2018-09-08
+Aura is the consensus engine pioneered by [Parity Technologies](https://parity.io) in 2016 and used to power the [public Kovan testnet](https://medium.com/@Digix/announcing-kovan-a-stable-ethereum-public-testnet-10ac7cb6c85f) which was established after the [Ropsten spam-attacks](https://ethereum.stackexchange.com/questions/12477/ropsten-testnet-is-under-kind-of-attack-what-can-we-do) in 2017. Besides, public value-bearing networks, such as the [POA Network](https://poa.network/), are based on the Aura protocol today.
 
 This document is _informational_ to summarize all available technical specification of the consensus engine to allow other client developer teams an overview, a place for feedback, and potentially the foundation to implement this in other code bases.
 
 ## Motivation
-Networks not bearing any value can not be sufficiently secured with proof-of-work or proof-of-stake. This is especially true for test networks that are supposed to be a reliable platform for developers to deploy and test their applications. Therefore, a proof-of-authority scheme is proposed which provides sufficient resistance against network attacks in testnets without attached value.
+Networks not bearing any value cannot be sufficiently secured with proof-of-work or proof-of-stake. This circumstance is uniquely right for test networks that are supposed to be a reliable platform for developers to deploy and test their applications. Therefore, a proof-of-authority scheme is proposed which provides sufficient resistance against network attacks in testnets without any value attached.
 
 ## Specification
-The following sections technically specifies the Authority Round consensus protocol.
+The following sections technically specify the Authority Round consensus protocol.
 
 ### Parameters
   - `n`, the number of validator nodes
   - `f`, the number of faulty validator nodes
   - `s`, the step ID
   - `t`, the step duration in seconds
+  - `h`, the best block height
+  - `C[K]`, the chain `C` at some denoted point `K`
 
 ### Authority Round Description
 Time is divided into discrete steps of duration `t`, determined by:
 
 ![Formula 00](../assets/eip-aura/00-unix-time.png)
 
-At each step `s`, a _primary_ will be assigned. Only the primary at a step may issue a block. It is misbehavior to produce more than one block per step or to produce a block out of turn.
-
-The primary for a step `s` is the node with index:
+At each step `s`, a _primary_ is assigned. Only the primary at a step may issue a block. It is misbehavior to produce more than one block per step or to produce a block out of turn. The primary for a step `s` is the validator node with index:
```

Comment:
3rd sentence - Producing more than one block per step or to produce a block out of turn is bad behavior.